instrumentation/instaawssdk/instaawssdk.go

```diff
@@ -96,5 +108,44 @@ func injectTraceContext(sp opentracing.Span, req *request.Request) {
 			opentracing.TextMap,
 			SNSMessageAttributesCarrier(params.MessageAttributes),
 		)
+	case *lambda.InvokeInput:
+		lc := LambdaClientContext{}
+
+		if params.ClientContext != nil {
+			res, err := base64.StdEncoding.DecodeString(*params.ClientContext)
```

Comment:
Since we have a serialize method defined for `LambdaClientContext`, would it make sense to add one to deserialize as well, to keep the logic close to each other.

Reply:
👍